modules/appeals_api/app/models/appeals_api/higher_level_review/contestable_issue.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class AppealsApi::HigherLevelReview::ContestableIssue
+  def initialize(issue)
+    @issue = issue
+  end
+
+  def decision_date
+    return unless decision_date_string
+
+    AppealsApi::HigherLevelReview::Date.new(decision_date_string)
+  end
+
+  def decision_date_string
+    issue.dig('attributes', 'decisionDate').to_s
+  end
+
+  def soc_date
+    return unless soc_date_string
+
+    AppealsApi::HigherLevelReview::Date.new(soc_date_string)
+  end
+
+  def soc_date_string
+    issue.dig('attributes', 'socDate')
```

Comment:
Curious why the `decisionDate` has a `#to_s` and the `socDate` doesn't...

Reply:
...I'm not sure 😅